### PR TITLE
Move sass files to static/styles

### PR DESCRIPTION
**Pull request by RubenRBS**
Also hides the tab titles on text overflow

### CODE REVIEW

File: static/compiler-service.ts [hunk 1]
```diff
@@ -429,11 +431,9 @@ export class CompilerService {
         const stdout = result.stdout ?? [];
         // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
         const stderr = result.stderr ?? [];
-        // TODO: Make its own const variable at top of module?
```
Comment (mattgodbolt) [inline static/compiler-service.ts:432]: nice! (unrelated to this change?)

File: static/styles/explorer.scss [hunk 2]
```diff
@@ -570,6 +570,10 @@ kbd {
     height: 20px !important;
 }
 
+li.lm_tab.lm_active {
```
Comment (mattgodbolt) [inline static/styles/explorer.scss:573]: is this intended too?

**Review by mattgodbolt — COMMENTED** [review]
flyby comments! all good

File: static/compiler-service.ts (hunk 1 above)
Comment (RubenRBS) [reply static/compiler-service.ts:432]: Yes. Was doing a bunch of things at the same time and ran out of time to separate it into various PRs, sorry!

File: static/styles/explorer.scss (hunk 2 above)
Comment (RubenRBS) [reply static/styles/explorer.scss:573]: Yes. Again, should have been a different PR

File: static/compiler-service.ts (hunk 1 above)
Comment (mattgodbolt) [reply static/compiler-service.ts:432]: n oworries! Just making sure things that aren't meant to get in aren't slipping before they're ready! thanks!

**Comment by partouf** [comment]
this is now live

**Pull request merged**